### ISSUE TITLE
html

### ISSUE THREAD

**Issue by Nitingif** [Nitingif/C117#1]
<html>

<head>
  <meta charset="UTF-8">
  <title>Sketch Identification</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>

  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/addons/p5.dom.min.js"></script>

  <script src="https://unpkg.com/ml5@latest/dist/ml5.min.js" type="text/javascript"></script>
  
  
    
  <link rel="stylesheet" type="text/css" href="style.css">
</head>

<body>
  <center>
    
    <div id="myModal" class="modal fade">
      <div class="modal-content">
        <div class="modal-header">
        </div>
        <div class="modal-body">
          <img src="sketch1.png">
          <img src="sketch2.png">
        </div>
      </div>
    </div>
    <h1>Sketch Identification
      <button id="show_sketch" class="btn btn-info" data-toggle="modal" data-target="#myModal">Show Sketch</button>
    </h1>
    <p id="label">Label : </p>
    <p id="confidence"> Confidence : </p>
    <button class="btn btn-danger btn-lg" onclick="clearCanvas()"> Clear Canvas </button>
  </center>

<script src="main.js"></script>
</body>
</html>
